docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -57,9 +57,9 @@
         <releaseinfo>API v1.0</releaseinfo>
         <productname>&PRODNAME;</productname>
         <!-- enter the publication date of this document; in revhistory, explain what changed -->
-        <pubdate>2014-12-04</pubdate>
+        <pubdate>2015-01-14</pubdate>
         <copyright>
-            <year>2014</year>
+            <year>2015</year>
```

Comment:
I'm not a lawyer, but if I recall correctly your options in the US are a) don't specify a year (which is fine) or b) specify all the years?